src/types/label.js

```diff
@@ -84,24 +84,12 @@ LabelAnnotation.defaults = {
     style: undefined,
     weight: undefined
   },
-  height: undefined,
   padding: 6,
   position: 'center',
   shadowBlur: 0,
   shadowOffsetX: 0,
   shadowOffsetY: 0,
   textAlign: 'center',
-  width: undefined,
-  xAdjust: 0,
```

Comment:
The same for x/yAdjust. Why?

Reply:
The xAdjust should not be removed. Not sure why any tests did not fail.